js-weather.js

```diff
@@ -0,0 +1,81 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const targetCity = document.getElementById('city');
+  const appId = '4b5774e9f3d2a07b84f0f2f88e486224';
+
+  const WeatherGetAsync = () => {
```

Comment:
```suggestion
  const getWeatherAsync = () => {
```
JSの慣例でアッパーキャメルのトークンはクラスやコンストラクタなので、ローワーキャメルにしましょう。動詞から始まるのも慣例です